processor/processorBenchmark_test.go

```diff
@@ -13,8 +13,9 @@ import (
 )
 
 func Benchmark_makeCommonMetadataFromSingularEvent(b *testing.B) {
```

Comment:
What's the difference in benchmark? Although in isolation isolation rlocks won't have much impact

Reply:
This was added a couple years ago. When we changed some json parsing logic inside. No intention to benchmark anything now.

Reply:
just out of curiosity, is there any observable difference if you run the same benchmark before and after the change?